app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/ContractExt.kt

```diff
@@ -0,0 +1,59 @@
+package com.hedvig.app.feature.insurance.ui.detail
+
+import com.hedvig.android.owldroid.graphql.InsuranceQuery
+import com.hedvig.app.R
+import com.hedvig.app.feature.insurance.ui.detail.yourinfo.YourInfoModel
+import com.hedvig.app.util.apollo.stringRes
+
+fun InsuranceQuery.Contract.toModelItems(): List<YourInfoModel> = when {
```

Comment:
How come we switched from the Row-based architecture to the more compounded structure here? 😄

Reply:
I think its difficult to find the right abstraction level for recyclerviews with different view types. For `YourInfoModel.Home` we should even use the new contract extension field, `currentAgreementDetailsTable()` (i'll create another PR for that). So I suppose we need to use a more Row-based architecture then :)